src/main/java/net/dv8tion/jda/api/requests/restaction/InviteAction.java

```diff
@@ -121,71 +121,62 @@
     @CheckReturnValue
     InviteAction setUnique(@Nullable final Boolean unique);
 
-    /**
-     * Sets the type of target for this voice channel invite.
-     *
-     * @param type
-     *        The target type for this voice channel invite.
-     *
-     * @throws java.lang.IllegalArgumentException
-     *         If this channel is not a voice channel.
-     *
-     * @return The current InviteAction for chaining.
-     */
-    @Nonnull
-    @CheckReturnValue
-    InviteAction setTargetType(@Nullable final Invite.TargetType type);
-
     /**
      * Sets the id of the targeted application.
-     * This will automatically {@link #setTargetType(Invite.TargetType) set the target type} to {@link Invite.TargetType#EMBEDDED_APPLICATION}.
+     * The invite has to point to a voice channel.
+     * The invite will have the {@link Invite.TargetType#STREAM} target.
```

Comment:
Is it correct that this was changed from EMBEDDED_APPLICATION to STREAM?

Reply:
Uhm no, done this by accident, copy pasting is bad